client/routes.js

```diff
@@ -21,13 +22,24 @@ class Routes extends Component {
   }
 
   render() {
-    const { isLoggedIn } = this.props
+    const { isLoggedIn, userIsAdmin } = this.props
 
     return (
       <Switch>
         {/* Routes placed here are available to all visitors */}
         <Route path="/login" component={Login} />
         <Route path="/signup" component={SignUpForm} />
+        {isLoggedIn && userIsAdmin && (
+          <Switch>
+            {/* Routes placed here are only available to logged in admins */}
+            <Route path="/home" component={Home} />
```

Comment:
are these other routes actually needed here if they're covered below in lines 44-50?